finish/README.txt

```diff
@@ -25,50 +23,42 @@ BankService.java file also contains code that automatically fails the first 2
 requests that are made to the checkBalance service so as to demonstrate the states 
 of a circuit breaker. 
 
-The @Fallback annotation and the fallback method are in a separate file, 
-BankServiceWithFallback.java. The fallback method is invoked whenever a request 
-to the service fails or when the circuit is open. When the fallback is running 
-the method immediately returns the following message: 
-	"The last known balance of your account is $10,293". 
+The @Fallback annotation and the fallback method are demonstrated in the same source 
+file. The fallback method is invoked whenever a request to the service fails or when 
+the circuit is open. When the circuit is opened, the request is not allowed to go 
+through and the fallback is called immediately. 
```

Comment:
The sentence doesn't seem correct to me and I've a hard time with the wording "When the fallback is running". There are two conditions the fallback is called.
1. When the checkBalance is called and returned an exception. With that, the fallback is called to return the last known balance. The fallback will not be called until after processing through the service in this case.
2. When the circuit is in open state, the fallback is called immediately.

Does the sentence apply to both conditions? If it does, I would take out the word "immediately". The other reason I take out this sentence is because the previous paragraph gives a general flow on the circuit breaker part of the codes without going into specific on the balance output. I feel like the balance output won't have to be included in the fallback part of the codes. 

Let me know what you think after this explanation. Thanks.